lib/fluent/supervisor.rb

```diff
@@ -203,14 +259,33 @@ def options
       }
     end
 
+    def main_run
```

Comment:
Currently, `start` is for supervised configuration, and `main_run` is for execution without supervisor. But these names don't express it.
Can you consider better names for these methods?